src/main/java/alfio/controller/api/admin/CheckInApiController.java

```diff
@@ -199,12 +215,14 @@ public ResponseEntity<AttendeeSearchResults> searchAttendees(@PathVariable("publ
     public List<FullTicketInfo> findAllTicketsForAdminCheckIn(@PathVariable("eventId") int eventId,
                                                               @RequestBody List<Integer> ids,
                                                               Principal principal) {
+        accessService.checkEventMembership(principal, eventId, AccessService.CHECKIN_ROLES);
         validateIdList(ids);
         return checkInManager.getAttendeesInformation(eventId, ids, principal.getName());
     }
 
     @GetMapping("/check-in/{eventName}/label-layout")
     public ResponseEntity<LabelLayout> getLabelLayoutForEvent(@PathVariable("eventName") String eventName, Principal principal) {
+        accessService.canAccessEvent(principal, eventName);
```

Comment:
canAccessEvent returns a result of type "EventAndOrganizationId". We can use that